docs/developer-docs/cost-estimations-and-examples.md

```diff
@@ -0,0 +1,176 @@
+# Cycles cost estimations and examples
+
+
+## Overview
+
+To get a rough estimate of how much your project may cost, below are a few common project architectures and their estimated monthly cost in cycles. These estimates are broken down into the following transaction groups:
+
+
+- **Messaging**: A message transaction refers to calls that are made to a canister's methods. Messaging costs depend on the type of call being sent (query, update, inter-canister, etc), the size of the message's request and response bytes, and the total amount of messages a canister sends and receives.
+
+- **Execution**: An execution transaction refers to tasks that a canister executes and the number of instructions executed per task and per call.
+
+- **Storage**: A storage transaction refers to the amount of data the canister stores.
+
+- **Special features**: Special feature transactions include HTTPS outcalls, transactions using the [Bitcoin API](./integrations/bitcoin/bitcoin-how-it-works.md), and transactions using the [chain-key signing API](./integrations/t-ecdsa/t-ecdsa-how-it-works.md).
+
+For detailed information on these different transaction types and what their contributing cost drivers are, please see the [cycles and transaction costs](cycles-and-transaction-costs.md) documentation page. It is recommended that you read that page as a prerequisite to reading this page.
+
+
+## Units and fiat value
+
+The price of cycles is fixed against the price of [XDR](/docs/current/references/glossary#xdr), where **1 trillion cycles equals 1 XDR**.
+
+This documentation will use the following units to measure and calculate the number of cycles and their associated fiat value:
+
+
+| Abbr. | Name | In numbers | Cycles XDR value | Cycles USD value |
+|---------------------------------------------------------------------------|
+| T | Trillion | 1_000_000_000_000 | 1 | 1.34 |
+| B | Billion | 1_000_000_000 | 0.001 | 0.00134 |
+| M | Million | 1_000_000 | 0.000001 | 0.00000134 |
+| k | Thousand | 1_000 | 0.000000001 | 0.00000000134 |
+| – | (one) | 1 | 0.000000000001 | 0.00000000000134|
+
+
+## How is the number of cycles charged to a canister computed?
```

Comment:
People might misunderstand this as "how IC actually charges canisters", but this is used only for our estimation and categorisation of the examples below. For example, IC doesn't change depending on the number of users. Charging in IC is done at lower-level: instructions, bytes, etc. The number of users is something that we use to estimate the low-level numbers.

This is kind of a higher-level, more developer facing pricing model that serves as bridge between the developer and the actual pricing model:

[developer]  <=> [this pricing model based on number of users] <=> [the actual IC pricing model based on low-level quantities].

Let's discuss this tomorrow over zoom?

Reply:
Changed the wording a bit to use 'estimated' instead of 'charged', and reflect that the following parameters are used for the estimation of cycles charges.